packages/react/src/components/MenuButton/_menu-button-shadow-blocker.scss

```diff
@@ -0,0 +1,192 @@
+@import '../../globals/vars';
```

Comment:
Should we use the new `@use` syntax for new sass files?

Reply:
Theoretically, I guess we could. I've just been adjusting it in the sass-module PR each time I update it. 🤷‍♂️

Reply:
@davidicus so, should we just take that decision then, that all new files use `@use`

Reply:
Since this old way is going to be deprecated I think it makes sense to avoid having to do so on a timeframe.

Reply:
Sounds good, I'll update the PR

Reply:
Nope, It doesn't work, it throws:
`Error: @use rules must be written before any other rules.`

Reply:
hmmm. Is it because this is pulled in somewhere else that is using imports? @kevinsperrine thoughts?